jobs/types/audit_errors.py

```diff
@@ -23,6 +23,8 @@
 from jobs import job_utils
 import python_utils
 
+ERROR_CATEGORY_COMMIT_CMD_CHECK = 'commit cmd check'
```

Comment:
I don't think this is needed. So please remove it.

Reply:
Done! @rohitkatlaa PTAL!